docs/using-hive.md

```diff
@@ -224,6 +256,26 @@ and replace the contents of `platform` with:
     region: us-east1
 ```
 
+For oVirt, replace the contents of `compute.platform` and `controlPlane.platform` with empty brackets (`{}`).
+```yaml
+controlPlane:
+  platform: {}
```

Comment:
Can't this just be omitted instead?

Reply:
I was just following along with the install-config.yaml that openshift-install creates.

Reply:
The install-config generated by the installer contains default values that are not required.